zzk/service/hostregistry.go

```diff
@@ -273,12 +273,15 @@ func (h *HostRegistryListener) Spawn(cancel <-chan interface{}, hostid string) {
 							// Only reschedule services without address
 							// assignments.
 
-							h.handler.UnassignAll(h.poolid, hostid)
+							err := h.handler.UnassignAll(h.poolid, hostid)
+							if err != nil {
+								logger.WithError(err).Warn("Got an error while unassigning services.")
+							}
 
 							count := DeleteHostStatesWhen(h.conn, h.poolid, hostid, func(s *State) bool {
 								return s.DesiredState == service.SVCStop || !s.Static
 							})
-							logger.WithField("unscheduled", count).Warn("Host is experiencing an outage.  Cleaned up orphaned nodes")
+							logger.WithField("unscheduled", count).Warnf("Host is experiencing an outage.  Cleaned up orphaned nodes. Pool connection timeout: %d", h.getTimeout())
```

Comment:
That value can change if someone changes the configuration during an outage.  So you will want to get the timeout value when it is set here:
https://github.com/control-center/serviced/pull/3682/files#diff-055fe4368b27718d7d6269ec11f2cc8aR147